### DISCUSSION TITLE
Nao baixa videos da Hotmart

### DISCUSSION THREAD

**Discussion by valenv0307** [aclap-dev/video-downloadhelper#1045] [Help and Q&A]
### Your question

Nao consigo baixar videos da Hotmart com a minha licença do download helper, ja fiz isso antes varias vezes e funcionava normalmente, agora estou tendo problemas

https://hotmart.com/pt-br

### Addon & CoApp version and details

Nao consigo baixar videos da Hotmart com a minha licença do download helper, ja fiz isso antes varias vezes e funcionava normalmente, agora estou tendo problemas

https://hotmart.com/pt-br

**Comment by GrampaWildWilly** [comment]
Search this forum for other discussions of Hotmart.
